script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4_1.txt

```diff
@@ -10,9 +10,9 @@ I think it's at the end of the east wing..."
 <Page4>
 I'll have to ask Hisui or Kohaku-san next time.
 For now... well, let's go to the living room.
-They said Akiha would spend some time in the living room after dinner so she might still be there.
+They said Akiha would spend some time in there after dinner, so she might still be there.
 <Page5>
-"Oh. It's been a day, Shiki-kun. After a night, are you starting to feel like you're back in the Tohno residence?"
+"Oh. It's been a full day now, Shiki-kun. After staying overnight, are you starting to feel like you're back in the Tohno residence?" //This sentence makes sense but reads really bizarrely, gave an effort at making it read more naturally
```

Comment:
Staying overnight gives me mild _temporary_ stay vibes, maybe `Having spent the night here, are you...`? Or ignore me if nobody else reads it like this.